Examples/ObjectiveC/ExtrusionsExample.m

```diff
@@ -41,14 +40,14 @@ - (void)mapView:(MGLMapView *)mapView didFinishLoadingStyle:(MGLStyle *)style {
     layer.sourceLayerIdentifier = @"building";
     
     // Filter out buildings that should not extrude.
-    layer.predicate = [NSPredicate predicateWithFormat:@"extrude != false AND height >=0"];
```

Comment:
We _do_ want to extrude. https://github.com/mapbox/ios-sdk-examples/pull/86/#discussion_r116355227 was a suggestion to change this to `extrude = true`, not to remove the condition. By removing the condition, buildings like the Eiffel Tower will get extruded quite humorously.